api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageType.java

```diff
@@ -9,5 +9,7 @@
     DETERMINE_ENTITLEMENT,
     ADDITIONAL_PREGNANCY_PAYMENT,
     REPORT_CLAIM,
-    REPORT_PAYMENT
+    REPORT_PAYMENT,
+    REQUEST_PAYMENT,
```

Comment:
Presumably the last part of this will be to remove the `MAKE_PAYMENT` enum and processor?

Reply:
Correct, but I couldn't do it as it would break everything. So removing MAKE_PAYMENT will be a different pr